org-code-javabuilder/playground/src/main/java/org/code/playground/Board.java

```diff
@@ -185,6 +184,11 @@ public void start() throws PlaygroundException {
       if (message != null) {
         this.handleClickEvent(message);
       }
+      if (this.isRunning) {
+        // Only need to send update complete if the game is still running
+        this.playgroundMessageHandler.sendMessage(
```

Comment:
should this be sent if the message was null?

Reply:
Yeah, I was thinking the same - opted to do it here so that we always send back an update complete message even if somehow we got a blank playground message, so that click events can get re-enabled.